HelpSource/Classes/Pclump.schelp

```diff
@@ -0,0 +1,76 @@
+TITLE:: Pclump
+summary:: Clumps the pattern into arrays of size n.
+categories:: Streams-Patterns-Events>Patterns>Repetition
+
```

Comment:
could have a `related` tag pointing e.g. at the `SequenceableCollection`'s `clump` method (don't recall the syntax for this...

Reply:
I can try. I think you can only point to a class.